tests/fixtures.py

```diff
@@ -10,13 +8,18 @@
 @fixture(name="dvc_repo_session", scope="session")
 def fix_dvc_repo_session() -> DvcRepo:
     with tempfile.TemporaryDirectory() as dir:
-        # os.mkdir("./temp_test")
         subprocess.check_call(["git", "init"], cwd=dir)
+        print(dir)
         dvc = DvcRepo.init(dir, subdir=True)
         yield dvc
         dvc.close()
 
 
 @fixture(name="dvc_repo")
-def fix_dvc_repo() -> DvcRepo:
-    return fix_dvc_repo_session()
+def fix_dvc_repo(dvc_repo_session) -> DvcRepo:
+    subprocess.check_call(
+        f"cp -r {dvc_repo_session.root_dir} {dvc_repo_session.root_dir}_1",
+        shell=True,
+    )
+    dvc_repo_session.root_dir += "_1"
```

Comment:
4. Ah -- and also, we need to create a new Repo so that internal state is fresh for each new test.

Reply:
Not sure I'm interpreting this entirely correctly, but I made a change to do a deepcopy so it's not potentially overriding the existing Repo